ivy/functional/backends/paddle/experimental/creation.py

```diff
@@ -16,12 +16,12 @@
 # Code from cephes for i0
 
 
-def _kaiser_window(window_length, beta):
-    n = paddle.arange(0, window_length)
-    alpha = (window_length - 1) / 2.0
-    return paddle_backend.i0(
-        beta * paddle.sqrt(1 - paddle_backend.divide((n - alpha), alpha) ** 2.0)
-    ) / paddle_backend.i0(beta)
+def _kaiser_window(M, beta):
+    Z = float((M - 1) / 2)
+    n = paddle.arange(-Z, Z + 1, step=1)
+    for_sqrt = ivy.sqrt(1 - ((4 * (n**2) / (M - 1) ** 2)))
+    ret = ivy.i0(beta * for_sqrt) / ivy.i0(beta)
+    return ret
```

Comment:
well for one that function didnt work, ivy.<function> basically runs the paddle backend in the back end no? that seemed to fix the paddle backend kaiser window and why the equation is changed is because the Numpy version of the equation basically gives the same result but more efficiently and for a larger range of beta values so when rewriting the backend to remove errors, I chose to change the equation to it instead (Alpha in certain edge cases can result in a non valid input to the i0() function

Reply:
It doesnt always fail if u use the paddle version but in certain cases it does, more so with the original equation implemented and very very rarely (once out of around the countless times I ran the tests) it failed for the current equation if I used the paddle backend, maybe it is more of a confirmation bias but I fixed it and ill upload the new commit